.storybook/preview.js

```diff
@@ -67,6 +68,22 @@ export const parameters = {
     // https://github.com/storybookjs/storybook/blob/v6.0.21/addons/docs/docs/docspage.md#inline-stories-vs-iframe-stories
     inlineStories: true,
     prepareForInline: (storyFn) => htmlToReactParser.parse(storyFn()),
+    transformSource(src, storyContext) {
+      try {
+        const storyFunction = storyContext.getOriginal();
+        const rendered = storyFunction(storyContext.args);
+        // The twing/source-inputs-loader.js file makes it so that whenever twig templates are rendered,
+        // the arguments and input path are stored in the window.__twig_inputs__ variable.
+        // __twig_inputs__ is a map between the output HTML and and objects with the arguments and input paths
+        // Here, since we have the rendered HTML, we can look up what the arguments and path were
+        // that correspond to that output
+        const input = window.__twig_inputs__?.get(rendered);
+        if (!input) return src;
```

Comment:
Are we confident that `input` will always have a `path` and `args`? If not, we may want to check for those here as well?

Reply:
Never mind, I just saw that we're defining those properties ourselves in `twing/source-inputs-loader.js`

Reply:
Yep!